ansible/plugins/inventory/adoptopenjdk_yaml.py

```diff
@@ -46,7 +46,8 @@
   # providers - validated for consistency
   'provider': ('azure', 'marist', 'osuosl', 'scaleway',
         'macstadium', 'macincloud', 'softlayer', 'spearhead',
-        'packet', 'linaro','digitalocean', 'ibm', 'godaddy', 'aws')
+        'packet', 'linaro','digitalocean', 'ibm', 'godaddy',
+        'aws', 'esxi')
```

Comment:
Should we make this `packet-esxi` to make it clearer what the provider is? (Arguably doesn't need the `-esxi` but I would definitely be happier if it said packet somewhere)

Reply:
updated